.github/workflows/migration-cli-e2e-test.yml

```diff
@@ -1,15 +1,25 @@
 name: Migration CLI E2E Test
 
 on:
-  push:
-    branches:
-      - master
   pull_request:
     branches:
       - master
 
 jobs:
+  check:
+    runs-on: ubuntu-latest
+    outputs:
+      migration: ${{ steps.filter.outputs.migration }}
+    steps:
+      - uses: dorny/paths-filter@v2
```

Comment:
strange that it works because on another brranch it fails due to missing git checkout step (I think)

https://github.com/facebook/docusaurus/pull/3907